Neptune.API/Controllers/OnlandVisualTrashAssessmentAreaController.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Neptune.API.Services;
+using Neptune.API.Services.Authorization;
+using Neptune.EFModels.Entities;
+using Neptune.Models.DataTransferObjects;
+
+namespace Neptune.API.Controllers;
+
+[ApiController]
+[Route("onland-visual-trash-assessment-areas")]
+public class OnlandVisualTrashAssessmentAreaController(
+    NeptuneDbContext dbContext,
+    ILogger<OnlandVisualTrashAssessmentAreaController> logger,
+    KeystoneService keystoneService,
+    IOptions<NeptuneConfiguration> neptuneConfiguration)
+    : SitkaController<OnlandVisualTrashAssessmentAreaController>(dbContext, logger, keystoneService, neptuneConfiguration)
+{
+    [HttpGet]
+    [JurisdictionEditFeature]
+    public ActionResult<List<OnlandVisualTrashAssessmentAreaGridDto>> ListByStormwaterJurisdictionID()
+    {
+        var stormwaterJurisdictionIDs = People.ListStormwaterJurisdictionIDsByPersonID(DbContext, CallingUser.PersonID);
+        var onlandVisualTrashAssessmentAreaSimpleDtos = OnlandVisualTrashAssessmentAreas
+            .ListByStormwaterJurisdictionIDList(dbContext, stormwaterJurisdictionIDs).Select(x => x.AsGridDto()).ToList();
+        return Ok(onlandVisualTrashAssessmentAreaSimpleDtos);
+    }
+
+    [HttpGet("{onlandVisualTrashAssessmentAreaID}")]
+    [JurisdictionEditFeature]
+    public ActionResult<OnlandVisualTrashAssessmentAreaDetailDto> GetByID([FromRoute] int onlandVisualTrashAssessmentAreaID)
+    {
+        var areaDetailDto = OnlandVisualTrashAssessmentAreas.GetByID(DbContext, onlandVisualTrashAssessmentAreaID).AsDetailDto();

```

Comment:
I would like the variable names to match the returning dto; in this case, this should be var onlandVisualTrashAssessmentAreaDetailDto
if you use resharper rename, it will do it for you